doc/Overview/Reactive/in-apps.md

```diff
@@ -131,11 +130,64 @@ so you can enhance the UX of the pagination, like display a loading indicator wh
 </uer:FeedView>
 ```
 
+## Selection
+
+The reactive framework has embedded support of selected item of a `ListFeed` data-bound to a `Selector`, like the `ListView`.
+It means that you don't have to data-bind (and synchronize) the `ListView.SelectedItem`, it will instead be automatically pushed from and to the view.
```

Comment:
```suggestion
It is not required to data-bind (and synchronize) the `ListView.SelectedItem`, it will instead be automatically pushed from and to the view.
```